crypto/src/asset.rs

```diff
@@ -137,7 +144,7 @@ mod tests {
 
         assert_eq!(
             base_denom.best_unit_for(0u64.into()).to_string(),
-            "penumbra"
+            "upenumbra"
```

Comment:
This should be set back to `penumbra`, the tweak to `best_unit_for` seems to have been dropped, which is also causing the smoke tests to fail

Reply:
If it was recent, it may not have been part of my latest rebase, or got lost in a merge. Will investigate and revert this change